doc/releases/v03.02.adoc

```diff
@@ -4,35 +4,36 @@ https://www.youtube.com/watch?v=SJUhlRoBL8M[Think p(OSI)tive!]
+A new chapter in the development of OSI has been opened with the https://www.asam.net/conferences-events/detail/asam-osi-kick-off-workshop/[kickoff workshop], which will hopefully generate lots of project proposals to streamline the efficient future development of the interface.
+Whith this release, we therefore decided to focus on improving processes and documentation.
 
 == https://github.com/OpenSimulationInterface/open-simulation-interface/pulls?q=is%3Apr+is%3Aclosed+milestone%3Av3.2.0[Content/Changes]
 
 
-* Updated README according to K(eep) I(t) S(imply) S(tupid) principle
-* Added Issue and Pullrequest templates
-* Travis CI updated
-** Update to bionic 18.04
-** Added tests for building protobuf 2.6 and 3.0 in parallel
-** Bugfixed the display of doubled documentation
-* Fixed several typos in the comments
-* Added a centralized documentation in https://github.com/OpenSimulationInterface/osi-documentation[OSI-Documentation]
-** Added contributors guidelines
-** Added commenting style guidelines
-** Added uniform citation style according to APA
-** Updated installation guides for osi in Linux and Windows
-** Added a reference in sphinx based on the generated doxygen documentation 
-* Added rules to the comments in order to enforce osi message validation with the https://github.com/OpenSimulationInterface/osi-validation[OSI-Validator]
-* Added the rule definitions in the rules.yml
-* Modularized testing of commenting style into python unit tests
-* Defined a standardazied a file format for reading trace files (*.osi)
-* Defined naming conventions of osi files
-* Added a *.txt to *.osi file converter for backward compatibility
-* Added a script to make traces readable for plausibility checks
-* Corrected the citing style for OSI Bibtex
-* Corrected unit specifications and references according to DIN (e.g. removed Unit: [])
-* Added timestamp in osi_environment
-* Added is_out_of_service bool to traffic light / traffic sign
+* Updated README according to K(eep) I(t) S(imply) S(tupid) principle.
+* Added Issue and Pullrequest templates.
+* Updated Travis CI.
+** Updated to bionic 18.04.
+** Added tests for building protobuf 2.6 and 3.0 in parallel.
+** Bugfixed the display of doubled documentation.
+* Fixed several typos in the comments.
+* Added a centralized documentation in https://github.com/OpenSimulationInterface/osi-documentation[OSI-Documentation].
+** Added contributors guidelines.
+** Added commenting style guidelines.
+** Added a uniform citation style based on APA.
+** Updated installation guides for osi in Linux and Windows.
+** Added a reference in sphinx based on generated doxygen documentation.
+* Added rules to the comments to enforce osi message validation with the https://github.com/OpenSimulationInterface/osi-validation[OSI-Validator].
+* Added the rule definitions in the rules.yml.
+* Modularized testing of commenting style into python unit tests.
+* Defined a standardized file format for reading trace files (*.osi).
```

Comment:
@pasched So now we fix typos in the release notes? 😃

Reply:
@stefancyliax In the heat of the review battle I just fixed them all and did not think about the agreement to leave them as they were. Please ignore these changes. ;)